includes/html/table/alerts.inc.php

```diff
@@ -155,7 +156,11 @@
         $severity .= ' <strong>-</strong>';
     }
 
-    $hostname = '<div class="incident">' . generate_device_link($alert, shorthost($alert['hostname'])) . '<div id="incident' . ($rulei + 1) . '" class="collapse">' . $fault_detail . '</div></div>';
+    if (Config::get('force_ip_to_sysname')) {
```

Comment:
You can use `format_hostname()` to do this without an `if/else` block.

Reply:
I'll have to look into this more, at first glance it's no obvious to me how it would work. I'm trying to display both the hostname AND sysname at the same time. How it is displayed on the webgui is based on what **force_ip_to_sysname** is set to, it will flip-flop the placement of hostname and sysname. The exact same behavior can be seen in the devices list page. I'll look at that code and see if i can figure it out. thanks

Reply:
You're correct, you might need to create another helper function unless one exists already.

Reply:
@laf After looking into this more, it seems your converting most of the pages to laravel blades? However the alert notification page hasn't been done yet? I didn't want to put a ton of work into this if your just going to convert it to a blade later. If i can get a function working it seems like it would be useful to use it everywhere hostname is displayed to get it somewhat standardized across all pages. I noticed some pages display hostname + sysname, while others just display the hostname. 

I will say this PR has helped a ton in determining what is down at a glance if your just using IPs for the hostname.

Reply:
@djamp42 made the change for you ;)